tests/insert_license_test.py

```diff
@@ -152,27 +165,34 @@ def test_is_license_present(src_file_content, expected_index):
+            ('module_with_license.css', '/*| *| */', False, 'module_without_license.css', True, False),
             ('module_with_license_and_few_words.css', '/*| *| */', False,
-                'module_without_license_and_few_words.css', True),
-            ('module_with_license_todo.css', '/*| *| */', False, None, True),
-            ('module_with_fuzzy_matched_license.css', '/*| *| */', False, None, False),
-            ('module_without_license.css', '/*| *| */', False, None, False),
-
-            ('module_with_license.py', '#', False, 'module_without_license.py', True),
-            ('module_with_license_and_shebang.py', '#', False, 'module_without_license_and_shebang.py', True),
-            ('init_with_license.py', '#', False, 'init_without_license.py', True),
-            ('init_with_license_and_newline.py', '#', False, 'init_without_license.py', True),
+                'module_without_license_and_few_words.css', True, False),
+            ('module_with_license_todo.css', '/*| *| */', False, None, True, False),
+            ('module_with_fuzzy_matched_license.css', '/*| *| */', False, None, False, False),
+            ('module_without_license.css', '/*| *| */', False, None, False, False),
+
+            ('module_with_license.py', '#', False, 'module_without_license.py', True, False),
+            ('module_with_license_and_shebang.py', '#', False, 'module_without_license_and_shebang.py', True, False),
+            ('init_with_license.py', '#', False, 'init_without_license.py', True, False),
+            ('init_with_license_and_newline.py', '#', False, 'init_without_license.py', True, False),
             # Fuzzy match
-            ('module_with_license.css', '/*| *| */', True, 'module_without_license.css', True),
-            ('module_with_license_todo.css', '/*| *| */', True, None, True),
-            ('module_with_fuzzy_matched_license.css', '/*| *| */', True, 'module_with_license_todo.css', True),
-            ('module_without_license.css', '/*| *| */', True, None, False),
-            ('module_with_license_and_shebang.py', '#', True, 'module_without_license_and_shebang.py', True),
+            ('module_with_license.css', '/*| *| */', True, 'module_without_license.css', True, False),
+            ('module_with_license_todo.css', '/*| *| */', True, None, True, False),
+            ('module_with_fuzzy_matched_license.css', '/*| *| */', True, 'module_with_license_todo.css', True, False),
+            ('module_without_license.css', '/*| *| */', True, None, False, False),
+            ('module_with_license_and_shebang.py', '#', True, 'module_without_license_and_shebang.py', True, False),
+            # Strict and flexible years
+            ('module_with_stale_year_in_license.py', '#', False, None, False, False),
+            ('module_with_stale_year_range_in_license.py', '#', False, None, False, False),
+            ('module_with_license.py', '#', False, 'module_without_license.py', True, True),
```

Comment:
Same suggestion here: could you add a test with a badly formatted year range please?

Reply:
Done